include/ulfius.h

```diff
@@ -1553,6 +1553,17 @@ int ulfius_websocket_send_fragmented_message(struct _websocket_manager * websock
                                              const char * data,
                                              const uint64_t fragment_len);
 
+/**
+ * Sends a JSON message in the websocket
+ * @param websocket_manager the websocket manager to use for sending the message
+ * @param message the message to send
+ * @return U_OK on success
+ */
+#ifndef U_DISABLE_JANSSON
+int ulfius_websocket_send_json_message(struct _websocket_manager * websocket_manager,
+                                       const json_t * message);
```

Comment:
Better off leaving the `const` keyword. Sometimes the compiler is unhappy when you pass a `json_t *` and it expects a `const json_t *`...
Also, if it is a const pointer, only the pointer would be a constant, not its content

Reply:
Fixed.